crates/common/src/config.rs

```diff
@@ -6,6 +6,39 @@ use citrea_pruning::PruningConfig;
 use serde::de::DeserializeOwned;
 use serde::{Deserialize, Serialize};
 use sov_stf_runner::ProverGuestRunConfig;
```

Comment:
this would also be implemented in ProverGuestRunConfig

Reply:
similarly, implementing `FromEnv` for `ProverGuestRunConfig` and `PruningConfig` in their own crate causes cyclic dependency here, lmk if there is a better way of implementing these